src/lib/pickles/step_branch_data.ml

```diff
@@ -161,7 +161,30 @@ let create
         ~wrap_rounds:Backend.Tock.Rounds.n ~feature_flags
       (* TODO *)
     in
-    Fix_domains.domains
+    let lookup_table_length_log2 =
+      let { Plonk_types.Features.range_check0
+          ; range_check1
+          ; foreign_field_add = _
+          ; foreign_field_mul
+          ; xor
+          ; rot
+          ; lookup
+          ; runtime_tables
+          } =
+        actual_feature_flags
+      in
+      let total_length =
+        ( if range_check0 || range_check1 || foreign_field_mul || rot then
+          Int.pow 2 12
+        else 0 )
+        + (if xor then 3 else Int.pow 2 8)
+        + (if lookup then 0 (* TODO: Need more info. *) else 0)
+        + if runtime_tables then 0 (* TODO: Need more info. *) else 0
```

Comment:
How come we have parentheses in one line and not the other?  Could it be not `ocamlformat`-formatted?